CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [2.0.5] - 2022-MM-DD
 ### Added
+- Added Support for `normalization_resolver` in `MLP` ([#4951](https://github.com/pyg-team/pytorch_geometric/pull/4951))
```

Comment:
maybe we should add this to line 10 instead? the changelog is getting very big 😓

Reply:
Good idea!